src/Fantomas.Cmd/Program.fs

```diff
@@ -241,8 +247,35 @@ let main _args =
 
     ArgParser.Parse(options, handleInput, "Fantomas <input_path>")
 
+    let loadConfig path =
+        match FantomasConfig.tryFindAndLoadConfig path with
+        | Some (Ok config) -> 
+            printfn "Loaded config '%s'" config.FileName
+            config.Warnings |> Seq.iter (printfn "Warning! %s")
+            config.FormatConfig
+        | Some (Error err) -> 
+            printfn "Warning! %s" err
+            FormatConfig.Default
+        | None ->
+            printfn "Unable to locate config file"
+            FormatConfig.Default
+
+    let baseConfig =
+        match String.IsNullOrWhiteSpace !configFile with
+        | true ->
+            match !inputPath with
+            | InputPath.Unspecified -> FormatConfig.Default
+            | InputPath.StdIn _     -> FormatConfig.Default
+            | InputPath.Folder f    -> loadConfig f
+            | InputPath.File f      -> loadConfig f
+        | false ->
+            let config = FantomasConfig.load !configFile // this will throw an exception if there is something wrong
```

Comment:
IMO we should never throw exception from CLI, emit error and exit with non-zero exit code.